selftests/run_coverage

```diff
@@ -6,7 +6,9 @@
 # Author: Lukas Doktor <ldoktor@redhat.com>
 
 coverage erase
-coverage run --include "avocado/*" ./selftests/run
+rm .coverage.*
```

Comment:
In my host, there's no `.coverage.*`, while there's a `.coverage` file which is not removed by this command.